src/moscot/problems/cross_modality/_mixins.py

```diff
@@ -183,6 +187,60 @@ def cell_transition(  # type: ignore[misc]
             key_added=key_added,
         )
 
+    def annotation_mapping(  # type: ignore[misc]
+        self: CrossModalityTranslationMixinProtocol[K, B],
+        mapping_mode: Literal["sum", "max"],
+        annotation_label: str,
+        forward: bool,
+        source: str = "src",
+        target: str = "tgt",
+        scale_by_marginals: bool = True,
+        other_adata: Optional[str] = None,
+        cell_transition_kwargs: Mapping[str, Any] = types.MappingProxyType({}),
+    ) -> pd.DataFrame:
+        """Transfer annotations between distributions.
+
+        This function transfers annotation labels (e.g. cell types) between groups of cells.
```

Comment:
This function transfers annotations (e.g. cell type labels) between distributions of cells.